### ISSUE TITLE
iOS 64bit

### ISSUE THREAD

**Issue by AntonPieter** [StickSports/ANE-Can-Open-URL#4]
Hi,

I use CanOpenUrl in several apps. Can I still use it with iOS 64bit?

Thanks,

AP

**Comment by overlawled** [comment]
It will need to be rebuilt using the latest iOS SDK. @richardlord any chance of this happening in the very near future?

**Comment by richardlord** [comment]
Hi. I no longer work at Stick Sports, but my guess is this won't be updated until/unless Stick Sports need to update it for their own purposes. However, the repository includes a build script so someone else could build it for 64bit.

**Comment by quick6black** [comment]
i also use this, don't have a MAC to update :(

**Comment by kheftel** [comment]
I just got a mac, I could give it a try. I also use it.

**Comment by maylorbax** [comment]
if someone manage to build with 64-bit support, please, let me know!!

**Comment by gnanaprakash** [comment]
Yes, I could able to successfully build iOS extension targeting for 64bit arch. 

Here is the step that I followed
1. Load CanOpenUrlIosExtension iOS project into Xcode (6.x) (Note: You might get warning since this particular project was created using old version of Xcode) 
2. Modify Build Settings -> iOS Deployment Target to iOS 6.0
3. Modify Build Settings -> Architectures to Standard architectures (armv7, arm64)
4. Save the project & exit Xcode
5. Create and modify the build.config as per the instruction. (Note: There seem to be a typo for android.sdk property, so replace android.adk to android.sdk)

**Comment by quick6black** [comment]
gnanaprakash thanks for updating and providing instructions on how to do it

**Comment by Elegarret** [comment]
Hi @gnanaprakash ! Does your version contains working ANE that doesnt need to be built? Unfortunately I dont have Mac to build on my side... And I downloaded your branch, but failed to build app with error:

Undefined symbols for architecture armv7:
  "___divmodsi4", referenced from:
      __ZN12BlockedCodec12CalcGridSizeEiiPi in libRuntimeHMAOT.arm-air.a(ZlibCod
ec.o)
      __ZN12BlockedCodec11SetupBlocksEii in libRuntimeHMAOT.arm-air.a(ZlibCodec.
o)
  "___udivmodsi4", referenced from:
      __ZN11FlashString9AppendIntEli in libRuntimeHMAOT.arm-air.a(flashstring.o)

      __ZN7avmplus6String12hashCodeUIntEj in libRuntimeHMAOT.arm-air.a(StringObj
ect.o)
      _sqlite3BitvecSet in libRuntimeHMAOT.arm-air.a(sqlite3_all.o)
      _sqlite3BitvecTest in libRuntimeHMAOT.arm-air.a(sqlite3_all.o)
      _sqlite3BitvecClear in libRuntimeHMAOT.arm-air.a(sqlite3_all.o)
      __ZL10II_samplesP10mad_bitptrPK10quantclassPl in libRuntimeHMAOT.arm-air.a
(layer12.o)
      __ZL14scale_rationalmmm in libRuntimeHMAOT.arm-air.a(madtimer.o)
      ...
ld: symbol(s) not found for architecture armv7
Compilation failed while executing : ld64

**Comment by gnanaprakash** [comment]
Yes, You can find tested iOS Universal (armv7, arm64) version under https://github.com/CollabPlus/ANE-Can-Open-URL/tree/master/bin
